lib/install/management/appliance.go

```diff
@@ -596,8 +596,14 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 	},
 	)
 
+	// fix up those parts of the config that depend on the final applianceVM folder name
 	conf.BootstrapImagePath = fmt.Sprintf("[%s] %s/%s", conf.ImageStores[0].Host, d.vmPathName, settings.BootstrapISO)
 
+	if len(conf.ImageStores[0].Path) == 0 {
```

Comment:
Correct, it will error before this point if no default datastore is found (in the event the user doesn't supply one at create time).